src/com/fairphone/fplauncher3/SettingsActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.fairphone.fplauncher3;
+
+import android.os.Bundle;
+import android.preference.PreferenceActivity;
+
+public class SettingsActivity extends PreferenceActivity {
+
+    public static final String KEY_PREF_ICON_PACK = "pref_iconPack";
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        addPreferencesFromResource(R.xml.preferences);
```

Comment:
Android Studio marks this method as deprecated from API 16. Since we're on `minSdkVersion 17`, we should do this in another way.